docs/source/conf.py

```diff
@@ -53,8 +53,8 @@
 project = u'Iroha'
 documentation = u'Iroha Documentation'
 description = u'Distributed ledger technology platworm, written in C++'
```

Comment:
I know this is not part of this change, but `platform` is spelled incorrectly in this line.